packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -3203,6 +3203,11 @@ export function getExecuteFunctions(
 			): Promise<unknown> {
 				const node = this.getNode();
 				const nodeType = workflow.nodeTypes.getByNameAndVersion(node.type, node.typeVersion);
+				if (!nodeType) {
+					throw new ApplicationError('Unknown node type and version', {
```

Comment:
Elsewhere we use `Node type and version unknown`. We might want to create a new error and throw always that instead, so that errors with this message are consolidated in Sentry.

Reply:
Will update them.